github/github-accessors.go

```diff
@@ -5,6 +5,8 @@
 
 // Code generated by gen-accessors; DO NOT EDIT.
 
+// +build !coverage
```

Comment:
I hope it's going to be clear to people that they shouldn't rely on this build tag, that it's for internal coverage use only and we may change it in the future. It'd be a shame if we can't get rid of it later without breaking user programs.